Library/Formula/cake.rb

```diff
@@ -0,0 +1,46 @@
+class Cake < Formula
+  desc "Cake (C# Make) is a build automation system with a C# DSL."
+  homepage "http://cakebuild.net/"
+  url "https://www.nuget.org/api/v2/package/Cake/0.5.1"
+  version "0.5.1"
+  sha256 "e3620f9bb1466d47502aeb934868284e241877ad5f43531f898443d846bd7c0b"
+
+  # Xamarin (and possibly other products) will maintain a current mono installation
+  # so we may not need to actually install it, so only depend on it if necessary
+  depends_on "mono" => :recommended
+
+  def install
+    libexec.install Dir["*.dll"]
+    libexec.install Dir["*.exe"]
+    libexec.install Dir["*.xml"]
+
+    # write a simple bash wrapper to execute `mono Cake.exe [args]` with the cake alias
+    bin.mkpath
+    (bin/"cake").write <<-EOS.undent
+      #!/bin/bash
+      mono #{libexec}/Cake.exe "$@"
+    EOS
+  end
+
+  test do
+    # just run cake to see the version, if we have an ok exit code it worked
+    system bin/"cake", "-version"
```

Comment:
Let's scrap this check. If the script below fails the test will exit as a failure.